tutorials/ar-subs/index.md

```diff
@@ -0,0 +1,795 @@
+
+1.  Using the SSH or console connection to the Pi, clone the repository
+associated with the Google Cloud community tutorials:
+
+        git clone https://github.com/GoogleCloudPlatform/community.git
+
+        NOTE! Googlers, use this during the review stage:
+        gcloud source repos clone ar-subs --project=solution-demo
+
+1.  Change to the tutorial directory:
+
+        community/tutorials/ar-subs
+
+1.  Create a Python 3 virtual environment:
+
+        python3 -m venv venv
+
+1.  Activate the virtualenv:
+
+        source venv/bin/activate
+
+1.  Upgrade pip and setuptools:
+
+        pip3 install -U pip setuptools
+
+1.  Install the required Python modules:
+
+        pip3 install -r requirements.txt
+
+
+### Create a Service Account and JSON key
+
+Next we will create a Service Account in your GCP project, and grant sufficient
+permissions to it, so that it can use the AI services. We will also need to
+download a JSON key for the Service Account. The JSON key will be used by the
+Python utilities to authenticate with the Cloud services.
+
+1. Create a new Service Account with the following command:
+
+        gcloud iam service-accounts create ml-dev --description="ML APIs developer access" --display-name="ML Developer Service Account"
+
+1. Grant the `ML Developer` role to the Service Account:
+
+        gcloud projects add-iam-policy-binding $PROJECT_ID --member serviceAccount:ml-dev@$PROJECT_ID.iam.gserviceaccount.com --role roles/ml.developer
+
+1. Grant also the Project Viewer role to the Service Account:
+
+        gcloud projects add-iam-policy-binding $PROJECT_ID --member serviceAccount:ml-dev@$PROJECT_ID.iam.gserviceaccount.com --role roles/viewer
+
+1. Create a JSON key for the Service Account. The key file will be downloaded to the current working directory:
+
+        gcloud iam service-accounts keys create ./credentials.json --iam-account ml-dev@$PROJECT_ID.iam.gserviceaccount.com
+
+
+### Identify your USB Microphone device number in python
+
+Next we need to identify the device number of your USB microphone visible to
+Python. Note that the device numbering for OS sound libraries may not match
+the device numbering visible to Python apps. For this reason we need to find
+the microphone device again, this time with a Python utility.
+
+1. Within the python virtual environment, execute the following command:
+
+        python3 mic_identify.py
+
+The command should output something similar to this - with the example USB
+microphone being the first entry listed here:
+
+
+    (0, 'Samson Meteor Mic: USB Audio (hw:2,0)', 2)
+    (1, 'dmix', 0)
+
+2. Note down the device number - in the above example it is: `0`
+
+
+### Test recording with the USB microphone in Python
+
+Let’s try to record audio with the identified USB microphone device,
+using `pyaudio`.
+
+1. Execute the following command. The command should record a 3 second WAV
+audio file (mono, 16bits, 44.1KHz). In this example we use the device `0` as
```

Comment:
Use 'you' instead of 'we' to address the reader, as you do in the rest of the tutorial

Reply:
done